app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1317,23 +1315,28 @@ class PatientRepositoryAndroidTest {
         creationFacilityUuid = facilityUuid,
         status = Scheduled
     )
-    val recentPatient2 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Cancelled
-    )
-    val recentPatient3 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Visited
-    )
```

Comment:
These are needed because we want to verify that only the right patients are fetched in the recent patients query.